public/app/plugins/datasource/influxdb/components/editor/config-v2/ConfigEditor.tsx

```diff
@@ -1,32 +1,37 @@
+import { css } from '@emotion/css';
 import React from 'react';
 
-import { Alert, Box, Stack, TextLink } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, Box, Stack, TextLink, useStyles2 } from '@grafana/ui';
 
 import { DatabaseConnectionSection } from './DatabaseConnectionSection';
 import { LeftSideBar } from './LeftSideBar';
 import { UrlAndAuthenticationSection } from './UrlAndAuthenticationSection';
+import { CONTAINER_MIN_WIDTH } from './constants';
 import { trackInfluxDBConfigV2FeedbackButtonClicked } from './tracking';
 import { Props } from './types';
 
 export const ConfigEditor: React.FC<Props> = ({ onOptionsChange, options }: Props) => {
+  const styles = useStyles2(getStyles);
   return (
     <Stack justifyContent="space-between">
-      <Box width="250px" flex="0 0 250px">
-        <LeftSideBar pdcInjected={options?.jsonData?.pdcInjected!!} />
-      </Box>
-      <Box width="60%" flex="1 1 auto">
+      <div className={styles.hideOnSmallScreen} style={{ width: '250px', flex: '0 0 250px' }}>
```

Comment:
Looks like the `style` props are also duplicated in the `styles.hideOnSmallScreen` object.

Reply:
I think you could also use `const { width } = useWindowSize()` as you have above to achieve this, without needing the `styles.hideOnSmallScreen` object.

Reply:
i removed the duplicate styles - thanks!